tests/test_config.py

```diff
@@ -147,7 +150,8 @@ def test_unknown_format():
 def test_no_such_template():
     # If you specify a template name, and it doesn't exist, an error will
     # be raised.
-    with pytest.raises(Exception, match="No such file: changelog.d/foo.j2"):
+    match = "No such file: changelog.d" + getattr(os.path, "sep", "") + "foo.j2"
+    with pytest.raises(Exception, match=re.escape(match)):
```

Comment:
How would you feel about:
```
with pytest.raises(Exception, match=r"No such file: changelog\.d[/\\]foo\.j2"):
```

Reply:
Sound good to me!

Reply:
Hmm, that puts me three characters over the limit. Black wants to reformat this as:

```diff
-    with pytest.raises(Exception, match=r"No such file: changelog\.d[/\\]foo\.j2"):
+    with pytest.raises(
+        Exception, match=r"No such file: changelog\.d[/\\]foo\.j2"
+    ):
```

How should I proceed?

Reply:
What black wants, black gets. :)

Reply:
Though you could also go with:
```
msg = r"No such file: changelog\.d[/\\]foo\.j2"
with pytest.raises(Exceptions, match=msg):
```

Reply:
Done.